concent_api/utils/tests/test_helpers.py

```diff
@@ -52,3 +57,61 @@ def test_parse_timestamp_to_utc_datetime_should_return_utc_datetime(self):
                 parse_timestamp_to_utc_datetime(timestamp),
                 expected_datetime
             )
+
+
+# pylint: disable=R0201
```

Comment:
Could this be changed into name instead of code ?

Reply:
done